src/scripts/main.js

```diff
@@ -2,6 +2,24 @@
 
 const pushNotification = (posTop, posRight, title, description, type) => {
   // write code here
+  const body = document.querySelector('body');
+  const block = document.createElement('div');
+  const hTwo = document.createElement('h2');
+  const paragraph = document.createElement('p');
+
+  hTwo.className = 'title';
+  hTwo.innerHTML = `${title}`;
+  paragraph.innerHTML = `${description}`;
+
+  block.className = `notification ${type}`;
+  block.append(hTwo);
+  block.append(paragraph);
+  body.append(block);
+
+  block.style.top = posTop + 'px';
+  block.style.right = posRight + 'px';
+
+  // setTimeout(() => block.remove(), 2000);
```

Comment:
```suggestion
  setTimeout(() => block.remove(), 2000);
```
Чому цей рядок закоментовано? По умові задачі, блок має зникати через дві секунди